src/time_integration/methods_SSP.jl

```diff
@@ -26,12 +26,14 @@ The third-order SSP Runge-Kutta method of Shu and Osher.
 struct SimpleSSPRK33{StageCallbacks} <: SimpleAlgorithmSSP
     a::SVector{3, Float64}
     b::SVector{3, Float64}
+    denom::SVector{3, Float64}
```

Comment:
Is this the same name as used in OrdinaryDiffEq? Otherwise I suggest to not abbreviate the name, as its meaning is (to me) not obvious until I see it in action
```suggestion
    denominator::SVector{3, Float64}
```